### PR TITLE
Fix losses

### PR DESCRIPTION
**Pull request by grenlayk**

### CODE REVIEW

**Review by SphericalPotatoInVacuum — APPROVED** [review]

File: src/training/stylegan_adversarial.py [hunk 1]
```diff
@@ -110,33 +110,35 @@ def train(self):
...
             reconstructed = self.model_G(style_content_embeds, style_embeds)
             reconstructed_loss = self.cons_loss(style_imgs, reconstructed)
 
             reconstructed_style_embeds = self.style_embedder(reconstructed)
-            cycle = self.model_G(style_content_embeds, reconstructed_style_embeds)
-            cycle_loss = self.cons_loss(style_imgs, cycle)
+            cycle_preds = self.model_G(style_content_embeds, reconstructed_style_embeds)
+            cycle_loss = self.cons_loss(style_imgs, cycle_preds)
+
+            ocr_loss_cycle, recognized = self.ocr_loss(cycle_preds, desired_labels, return_recognized=True)
```
[the remark this answers is not in the document]
Comment (nikoryagin) [reply src/training/stylegan_adversarial.py:135]: i dont think we need recognized content neither

File: src/training/stylegan_adversarial.py [hunk 2]
```diff
@@ -205,9 +207,9 @@ def validate(self, epoch: int):
             cycle = self.model_G(style_label_embeds, reconstructed_style_embeds)
             cycle_loss = self.cons_loss(style_imgs, cycle)
 
-            perc_loss, tex_loss = self.perc_loss(style_imgs, res)
-            emb_loss = self.typeface_loss(style_imgs, res)
-            adv_loss = self.model_G_adv_loss(res)
+            perc_loss, tex_loss = self.perc_loss(style_imgs, preds)
+            emb_loss = self.typeface_loss(style_imgs, preds)
+            adv_loss = self.model_G_adv_loss(preds)
```
Comment (nikoryagin) [inline src/training/stylegan_adversarial.py:212]: dont we want to calculate g loss with reconstructed images?

Comment (grenlayk) [reply src/training/stylegan_adversarial.py:212]: yes u r right, my bad

**Review by nikoryagin — APPROVED** [review]
